### ISSUE TITLE
training loss

### ISSUE THREAD

**Issue by qingzhong1** [yyyyyxie/textflux#19]
Hello, I was wondering if it's possible to disclose the loss from the beta training phase. I'm currently working on reproducing this project and would greatly appreciate a reference point to verify if my reproduction process is on the right track.

**Comment by yyyyyxie** [comment]
The loss curve stays basically the same, but the results actually get better the longer you train.
